netbox/virtualization/tables/virtualmachines.py

```diff
@@ -84,10 +86,17 @@ class VirtualMachineTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable)
     interface_count = tables.Column(
         verbose_name=_('Interfaces')
     )
+    virtual_disk_count = tables.Column(
+        verbose_name=_('Virtual Disks')
+    )
     config_template = tables.Column(
         verbose_name=_('Config Template'),
         linkify=True
     )
+    disk = tables.Column(
```

Comment:
If my congestion above is implemented, we could replace this with `virtual_disk_space`.